build.xml

```diff
@@ -39,7 +39,7 @@
     <property name="reports" value="dist/test"/>
     <property name="test.output" value="dist/test"/>
     <property name="htsjdk_lib_dir" value="${dist}/htsjdk_lib_dir"/>
-    <property name="htsjdk_git_url" value="git@github.com:samtools/htsjdk.git"/>
+    <property name="htsjdk_git_url" value="https://github.com/samtools/htsjdk.git"/>
```

Comment:
oops, this is not needed. I needed it only for my system. Will delete this change